test/adaptive_load/fake_plugins/fake_input_variable_setter/fake_input_variable_setter.proto

```diff
@@ -0,0 +1,15 @@
+syntax = "proto3";
+
+package nighthawk.adaptive_load;
+
+import "google/rpc/status.proto";
+
+// Configuration for FakeInputVariableSetter (plugin name: "nighthawk.fake_input_variable_setter")
+// that sets the |connections| field in CommandLineOptions.
+message FakeInputVariableSetterConfig {
+  // A factor that multiplies the input value before applying it, in order to test the propagation
+  // of both the input and the config. Optional, default 1.
+  int32 adjustment_factor = 1;
```

Comment:
Consistency nit: I think we've been mostly using int32_t in other code

Reply:
In proto we only seem to have int32. Also changing this to uint32 since we don't need negative values. Previously the artificial validation errors were generated by setting a negative adjustment_factor, but now we set the artificial error explicitly.

Reply:
Oh my bad, this is a proto and not c++. Sorry about the noise